internal/extsvc/github/common.go

```diff
@@ -1837,6 +1833,16 @@ type Repository struct {
 	Visibility Visibility `json:",omitempty"`
 }
 
+// Repository is a GitHub repository.
+type Repository struct {
+	*BaseRepository
+	// This field will always be blank on repos stored in our database because the value will be different
+	// depending on which token was used to fetch it
+	ViewerPermission string // ADMIN, WRITE, READ, or empty if unknown. Only the graphql api populates this. https://developer.github.com/v4/enum/repositorypermission/
```

Comment:
You touched it 😁 We can also fix the doc formatting in this case.

```suggestion
	// This field will always be blank on repos stored in our database because the value will be different
	// depending on which token was used to fetch it.
	// ADMIN, WRITE, READ, or empty if unknown. Only the graphql api populates this. https://developer.github.com/v4/enum/repositorypermission/
	ViewerPermission string
```